src/main/scala/viper/gobra/translator/Names.scala

```diff
@@ -25,6 +25,24 @@ object Names {
     scala.util.hashing.MurmurHash3.stringHash(s).toHexString
   }
 
+  object InterfaceMethod {
```

Comment:
As a tiny tiny nitpick (feel free to ignore), I would have named this object `InterfaceMethodProxy`

Reply:
It is in the names object. So, I feel that proxy is redundant.